.rubocop.yml

```diff
@@ -4,6 +4,7 @@ require:
 AllCops:
   DisplayCopNames: true
   DisplayStyleGuide: true
+  NewCops: enable
   TargetRubyVersion: 2.4
```

Comment:
My understanding is that `TargetRubyVersion` can be inferred for app repos, because an app is deployed with a specific version of ruby (usually declared in the Gemfile). But for library code like socrates, we support multiple versions and so that inference can't be done. So we need to be explicit about the lowest version of ruby that this library supports.

If I remove the `TargetRubyVersion` line, rubocop defaults to 2.7. It is not smart enough to infer based on the `required_ruby_version` in the gemspec.